templates/careers/_search.html

```diff
@@ -141,17 +157,22 @@
 });
 </script>
 {% endif %}
-<div class="js-search-jobs-form u-hide">
-  <form class="p-search-box">
-    <label class="u-off-screen" for="search">Search</label>
-    <input type="search" id="search" class="p-search-box__input js-careers__search-input" name="search" placeholder="Search" required="" autocomplete="off">
-    <button type="reset" class="p-search-box__reset"><i class="p-icon--close">Close</i></button>
-    <button type="submit" class="p-search-box__button"><i class="p-icon--search">Search</i></button>
+
+<div class="js-search-jobs-form u-hide u-align--bottom" style="display: flex;">
+  <form action="/search" class="p-form p-form--inline p-form--search">
```

Comment:
Clicking "Search roles" takes me to https://canonical-com-605.demos.haus/search?search=, which is a 404.